zephyr/wrapper.c

```diff
@@ -72,18 +72,32 @@ __section(".heap_mem") static uint8_t __aligned(64) heapmem[HEAPMEM_SIZE];
 
 #else
 
-#define HEAPMEM_SHARED_SIZE	(HEAP_SYSTEM_SIZE + HEAP_RUNTIME_SIZE + \
-				HEAP_RUNTIME_SHARED_SIZE + HEAP_SYSTEM_SHARED_SIZE)
+#if (CONFIG_HP_MEMORY_BANKS <= 8)
+/* APL */
```

Comment:
I'd remove these comments and add one "/* TODO: heap size should be set based on linker output */" instead at top.